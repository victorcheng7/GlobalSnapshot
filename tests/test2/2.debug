Finish Set Up!
Received a Money Transfer from 1
Received Done from Site 1 Originated from Site 1
Sending Done to Site 1
Sending Done to Site 3
Received a Money Transfer from 3
Sending Money to Site 1
Received Done from Site 3 Originated from Site 1
Sending Marker to Site 1
Sending Marker to Site 3
Received a MARKER from 1
Sending Money to Site 3
Sending Done to Site 1
Sending Done to Site 3
Received Done from Site 1 Originated from Site 2
Received a MARKER from 3
Snapshot Result: 2.1: 21 0 0
Received Done from Site 3 Originated from Site 2
Received Done from Site 3 Originated from Site 3
Sending Done to Site 1
Sending Done to Site 3
Shutting Down!
