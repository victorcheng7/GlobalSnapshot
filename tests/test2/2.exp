2.1: 7 5 9
or
2.1: 21 0 0
or
2.1: 12 0 9
or
2.1: 16 5 0
