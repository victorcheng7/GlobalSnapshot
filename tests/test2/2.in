send 1 3
snapshot
send 3 1
