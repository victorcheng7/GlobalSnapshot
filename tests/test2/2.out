2.1: 21 0 0
